README.md

```diff
@@ -29,42 +29,48 @@ To update the MHKiT documentation you will need Python 3.6+, Sphinx, and a clone
    **NOTE:** You may need to add PROXY settings ([see info here](https://cinhtau.net/2018/04/16/python-proxy-windows/))
 
   - Use the command line to install the needed Sphinx submodules (BibTex, MATLAB theme, rtd theme, NB Sphinx, and NB Sphinx Lin)
-   ``pip install -U sphinxcontrib-bibtex sphinxcontrib-matlabdomain sphinx_rtd_theme nbsphinx nbsphinx_link``
+  
+   ```pip install -U sphinxcontrib-bibtex sphinxcontrib-matlabdomain sphinx_rtd_theme nbsphinx nbsphinx_link```
 
 
 You are now ready to begin modifying and building the MHKiT documentation.
 
 ## Step 3. Modify the Documentation
 
 - The restructured text files used to build the documentation are located in the ``/MHKiT/docs/source`` directory (Check the [Sphinx Website](http://www.sphinx-doc.org/en/master/) for information about the folder structure).
-- Use a text editor to modify any restructured text file (files with `.rst` extension, e.g. `index.rst`).
+- Before modifying the documentation we reccomend creatting a feature branch and not modifying your fork's master branch. A feature branch can be created using the commands
+```bash
+git branch featureBranchName
+git checkout featureBranchName
+```
+- Now you can use a text editor to modify any restructured text file (files with `.rst` extension, e.g. `index.rst`).
 - Once you are done editing, move to Step 4
 
 ## Step 4. Build the [MHKiT Documentation](https://MHKiT-Software.github.io/MHKiT/)
 - To locally build the documentation use the command line to move into the ``/MHKiT/docs`` folder then `make html`
-```
+```bash
```

Comment:
When I type bash into the PowerShell I get this error:
"Windows Subsystem for Linux has no installed distributions.
Distributions can be installed by visiting the Microsoft Store:
https://aka.ms/wslstore"

Reply:
Why are you typing Bash into the PowerShell? 
``` means the start of a code block
bash means when rendering interpret as this coding language. It could be python, matlab etc.
If you look at this code block in the rendered view you will see that there is no "bash"